src/components/container/container.js

```diff
@@ -251,9 +251,13 @@ export default class Container extends UIObject {
     this.trigger(Events.CONTAINER_CONTEXTMENU, this, this.name)
   }
 
-  setCurrentTime(time) {
-    this.trigger(Events.CONTAINER_SEEK, time, this.name)
-    this.playback.seek(time)
+  setCurrentTime(time, inseconds) {
```

Comment:
I already created another method called seekPercentage. 
In my PR, there are two public methods available seek(seconds) seekPercentage(percentage) and that setCurrentTime is a private method.